packages/components/src/highlight-cards/annual-highlight-cards.tsx

```diff
@@ -1,23 +1,131 @@
 import { comment, Icon, paragraph, people, postContent, starEmpty } from '@wordpress/icons';
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
+import ComponentSwapper from '../component-swapper';
+import ShortenedNumber from '../number-formatters';
 import CountComparisonCard from './count-comparison-card';
+
 import './style.scss';
 
+type AnnualHighlightCounts = {
+	comments: number | null;
+	likes: number | null;
+	posts: number | null;
+	words: number | null;
+	followers: number | null;
+};
+
+type AnnualHighlightsMobileProps = {
```

Comment:
Could we consolidate `AnnualHighlightsMobileProps` and `AnnualHighlightsStandardProps` to only `AnnualHighlightProps`?